components/net/http_loader.rs

```diff
@@ -1132,35 +1126,47 @@ async fn http_network_or_cache_fetch(
     };
 
     // Step 8.4: If Cross-Origin-Embedder-Policy allows credentials with request returns false, then
-    // set includeCredentials to false.
+    //           set includeCredentials to false.
     // TODO: Requires request's client object
 
-    // Step 8.5: Let contentLength be httpRequest’s body’s length, if httpRequest’s body is
-    // non-null; otherwise null.
-    let content_length_value = match http_request.body {
-        Some(ref http_request_body) => http_request_body.len().map(|size| size as u64),
-        // Step 8.7: If httpRequest’s body is null and httpRequest’s method is `POST` or `PUT`, then
-        // set contentLengthHeaderValue to `0`.
-        None => match http_request.method {
-            Method::POST | Method::PUT => Some(0),
-            _ => None,
-        },
+    // Step 8.5 Let contentLength be httpRequest’s body’s length, if httpRequest’s body is non-null;
+    //          otherwise null.
```

Comment:
Ah you're doing this indentation on purpose? Usually we don't, but why not(if test-tidy passes).